en/_posts/2016-04-20-marathon-health-checks.md

```diff
@@ -6,23 +6,23 @@ author: devatoria
 lang: en
 ---
 
-Health checks are useful to ensure that your container is running; and better, is running as expected. You can add multiple health checks per application. A health check can be of two types:
+Health checks are useful to ensure that your containers are running; and even better, are running as expected. You can add multiple health checks per application. A health check can be of two types:
 
 - HTTP
 - TCP
 
-The only difference is that the HTTP check is checking a path (using http://localhost:port/path), and is able to ignore some HTTP status code (informational ones, from 100 to 199).
+The only difference is that the HTTP check is given a path (using http://localhost:port/path), and is able to ignore some HTTP status codes (informational ones, from 100 to 199).
 
-#Configure a health check for a nginx application
+# Configure a health check for a nginx application
 
-In the previous tutorial, we ran a simple nginx application. Let's add some health check for this application.
+In the [previous tutorial](https://community.runabove.com/kb/en/docker/quick-start-with-marathon.html), we ran a simple nginx container. Let's now configure some HTTP health check for this application.
 
 ![Main interface](/kb/images/2016-04-20-marathon-health-checks/health_check.png)
 
-- **Grace Period**: the time to wait from the launch of the task where health check errors will be ignored *(useful if you have a task which is long to be healthy and do not want your health check to fail because of this)*
-- **Interval**: the time to wait before each health check
-- **Timeout**: the time to wait before a health check is considered as failed regardless of the result
+- **Grace Period**: health check failures are ignored within this number of seconds or until the task becomes healthy for the first time *(useful if you have a task with a long startup time, to avoid having the  health check failing because of this)*.
```

Comment:
reader question: is the container in the load balancer for the initial grace period or is unrelated ? Use-case: I'd want my container to be registered in the load balancer only when it is ready